internal/common/utils.go

```diff
@@ -46,3 +47,13 @@ func ObfuscateFileName(name string) string {
 
 	return fmt.Sprintf("%x.dat", h.Sum(nil))
 }
+
+func ToString(i int64) string {
+	return strconv.FormatInt(i, 10)
+}
+
+func ToInt64(s string) int64 {
+	i, _ := strconv.ParseInt(s, 10, 0)
+
+	return i
+}
```

Comment:
Can we not use `Atoi` and `Itoa` instead?
https://golang.org/pkg/strconv/

Reply:
Unfortunately no. `Itoa` and `Atoi` works with `int`, not `int64`. The necessity of `int64` goes back to FUSE implementations where the `inode`s are strictly declared as `int64` to be compatible with 32bit systems I think.